constraint-solver/src/quadratic_symbolic_expression.rs

```diff
@@ -332,14 +332,22 @@ impl<T: FieldElement, V: Ord + Clone + Hash + Eq + Display> QuadraticSymbolicExp
             if coeff.is_known_nonzero() {
                 // In this case, we can always compute a solution.
                 let value = self.constant.field_div(&-coeff);
-                ProcessResult::complete(vec![Effect::Assignment(var.clone(), value)])
+                ProcessResult::complete(vec![assignment_if_satisfies_range_constraints(
```

Comment:
Incomplete merge?

Reply:
These are the changes from #2708. It should go away after I merge `main`, which I'll do once I can run main on `powdr-openvm` again.